packages/token-sdk/src/repository.ts

```diff
@@ -192,8 +184,8 @@ export class TokenRepository {
     if (tag === undefined) {
       return this.mintMap.has(mintString);
     }
-    const tagSet = this.mintMap.get(mintString);
-    return tagSet?.has(tag) ?? false;
+    const entry = this.mintMap.get(mintString);
+    return entry?.tags.has(tag) ?? false;
```

Comment:
nit: If ``has = true``, the mint included in fetchAll seems intuitive. How about checking ``exists``.

Reply:
This is a great catch - thank you. I've corrected and added it to a test.